documentation/docs/opal-plus/features.mdx

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 3
+title: Advanced Features
+---
+
+OPAL+ has a number of advanced features extending the capabilities of the open-source OPAL.
+These features are available to OPAL+ users only.
+
+To apply for Permit OPAL+, [fill in the form available here](https://hello.permit.io/opal-plus)
+
+
+## Support and SLA
+
+OPAL+ provides dedicated support and a custom SLA to help you get the most out of your OPAL+ deployment.
+Reach out to us [on Slack](https://bit.ly/permit-slack) for more information.
+
+## Licensing Capabilities
+
+OPAL+ provides additional licensing capabilities to help you manage your OPAL+ deployment.
+Reach out to us [on Slack](https://bit.ly/permit-slack) for more information.
```

Comment:
I don't think we need the reach out on slack for this feature

Reply:
So what other information should I write about it?